src/test/java/com/google/sps/data/DatastoreScheduledInterviewTest.java

```diff
@@ -0,0 +1,155 @@
+            "user@company.org",
+            "user@mail.com");
+    assertEquals(storedScheduledInterview, copyScheduledInterview1);
+  }
+
+  // Tests whether all scheduledInterviews for a particular user are returned.
+  @Test
+  public void getsAllScheduledInterviews() {
+    tester.create(scheduledInterview1);
+    tester.create(scheduledInterview2);
+    tester.create(scheduledInterview3);
+    List<ScheduledInterview> result = tester.getForPerson("user@company.org");
+    ScheduledInterview copyScheduledInterview1 =
+        ScheduledInterview.create(
+            result.get(0).id(),
+            new TimeRange(
+                Instant.parse("2020-07-06T17:00:10Z"), Instant.parse("2020-07-06T18:00:10Z")),
+            "user@company.org",
+            "user@mail.com");
+    ScheduledInterview copyScheduledInterview2 =
+        ScheduledInterview.create(
+            result.get(1).id(),
+            new TimeRange(
+                Instant.parse("2020-07-06T19:00:10Z"), Instant.parse("2020-07-06T20:00:10Z")),
+            "user@company.org",
+            "user2@mail.com");
+
+    assertEquals(2, result.size());
+    assertEquals(copyScheduledInterview1, result.get(0));
+    assertEquals(copyScheduledInterview2, result.get(1));
+  }
+
+  // Tests deleting a user's scheduledInterview.
+  @Test
+  public void deletesScheduledInterview() {
+    tester.create(scheduledInterview1);
+    tester.create(scheduledInterview2);
+    List<ScheduledInterview> result = tester.getForPerson("user@company.org");
+    tester.delete(result.get(0).id());
+    Entity entity = datastore.prepare(new Query("ScheduledInterview")).asSingleEntity();
+    ScheduledInterview storedScheduledInterview = tester.entityToScheduledInterview(entity);
+    ScheduledInterview copyScheduledInterview2 =
+        ScheduledInterview.create(
+            storedScheduledInterview.id(),
+            new TimeRange(
+                Instant.parse("2020-07-06T19:00:10Z"), Instant.parse("2020-07-06T20:00:10Z")),
+            "user@company.org",
+            "user2@mail.com");
+    assertEquals(storedScheduledInterview, copyScheduledInterview2);
+  }
+
+  // Tests updating a user's scheduledInterview.
+  @Test
+  public void updatesScheduledInterview() {
+    tester.create(scheduledInterview1);
+    Entity entity = datastore.prepare(new Query("ScheduledInterview")).asSingleEntity();
+    ScheduledInterview previousStoredScheduledInterview = tester.entityToScheduledInterview(entity);
+    ScheduledInterview updatedStoredScheduledInterview =
+        ScheduledInterview.create(
+            previousStoredScheduledInterview.id(),
+            new TimeRange(
+                Instant.parse("2020-07-06T19:00:10Z"), Instant.parse("2020-07-06T20:00:10Z")),
+            "user@company.org",
+            "user3@mail.com");
+    tester.update(previousStoredScheduledInterview);
+    assertEquals(false, previousStoredScheduledInterview.equals(updatedStoredScheduledInterview));
```

Comment:
This should be more than a "they're different" check. Compare what is there after the update to what you would expect to be there.